src/lumigo_tracer/wrappers/http/http_parser.py

```diff
@@ -354,12 +418,18 @@ def parse_response(self, url: str, status_code: int, headers, body: bytes) -> di
         )
 
 
-def get_parser(url: str, headers: Optional[dict] = None) -> Type[Parser]:  # type: ignore[type-arg]
+def get_parser(host: str, headers: Optional[dict] = None) -> Type[Parser]:  # type: ignore[type-arg]
+    # Headers are case-insensitive, so lets lowercase them and always search them in lowercase
+    lowercase_headers = {}
```

Comment:
probably dup with 
https://github.com/lumigo-io/python_tracer/blob/a48db8acfd1b353d0474809597c6be9f271ead96/src/lumigo_tracer/wrappers/http/sync_http_wrappers.py#L116

Reply:
Yes I saw this, but because the lowercase keys aren't promised in the function params I worry that the function can be called from somewhere else without lowercasing the keys...

What i will do is to  add a note to the functions docstring to make sure other people know lowercase the header keys.